lib/ids_please/grabbers/base.rb

```diff
@@ -3,30 +3,38 @@
 class IdsPlease
   module Grabbers
     class Base
-
       def self.interact(links)
-        links.map { |l| self.new(l).grab_link }
+        links.map do |l|
+          new(l).grab_link
+        end
       end
 
-      attr_reader :avatar, :display_name, :username, :link, :page_source, :network_id, :data, :counts
+      attr_reader :avatar,
+                  :display_name,
+                  :username,
+                  :link,
+                  :page_source,
+                  :network_id,
+                  :data,
+                  :counts
 
       def initialize(link)
         @link = link
       end
 
-      def grab_link(link)
+      def grab_link(_link)
         throw 'Base grabber can not grab anything'
       end
 
       def to_s
         line = ''
-        self.instance_variables.each do |iv|
```

Comment:
Style preferences for using metaprogramming methods as `:send` and `:instance_variables_get/set` with `self